src/components/table/OcTable.vue

```diff
@@ -39,7 +39,6 @@
     <tfoot v-if="$slots.footer" class="oc-table-footer">
       <tr>
         <td :colspan="footerColspan" class="oc-table-footer-cell">
-          <!-- @slot Footer of the table -->
```

Comment:
Why remove this line? It's used for the generated ODS docs. ;-)

Reply:
Felt more like a stray line since it's the only comment in the component template (there's 3 slots inside), but happy to revert the deletion if wanted!

Reply:
Reverted this line, let's merge this PR into master since the `6.0.2` release (branch) will be a dead end?